### DISCUSSION TITLE
DatePicker 日期选择框的分隔符看起来有些不协调

### DISCUSSION THREAD

**Discussion by xxxace** [vueComponent/ant-design-vue#7513] [General]
ant-picker-separator是单边上箭头，不知道是不是我的问题，整体看起来就好像是向上的，不是水平的。
为什么不用之前的2.0的~或者是→呢？

antd
![image](https://github.com/vueComponent/ant-design-vue/assets/60084325/b80d9cff-11a3-4af4-8ee4-d010be82b10a)
naiveui
![image](https://github.com/vueComponent/ant-design-vue/assets/60084325/353f4793-e9be-4cf2-b73d-5e0f8e0a15d3)

**Comment by xxxace** [comment]
或者可以放开让用户自定义吗？因为其它5个icon是可以自定义的，就这个分隔符没有

**Comment by ryne6** [comment]
![image](https://github.com/vueComponent/ant-design-vue/assets/91053541/3cf76969-04e1-483e-ab5f-9e6084a8fb53)

**Reply by xxxace** [reply]
非常感谢回答，这个确实是我的问题，我没看清楚文档，方便讨论下为什么你们会选择用单边箭头作为默认分隔符，从我个人的视觉感官来说如果排列在一起会有种不对称的感觉，当然每个人的感觉都不太相同，只是探讨一下。再次感谢你的回答

**Reply by ryne6** [reply]
不用客气 不过我不是官方人员 或许你可以看看ant-design的设计文档
